lib/sorbet-rails/model_plugins/active_record_attribute.rb

```diff
@@ -131,15 +131,23 @@ def time_zone_aware_column?(column_def, cast_type)
 
   sig { params(column_type: Object).returns(String) }
   def value_type_for_attr_writer(column_type)
```

Comment:
We probably should pass a `column_def` to this method and have a switcher based on `active_record_type` like `active_record_type_to_sorbet_type`